samples/matter/lock/sample.yaml

```diff
@@ -29,15 +29,23 @@ tests:
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
     tags: ci_build
-  sample.matter.lock.smp_dfu:
+  sample.matter.lock.smp_dfu_ffs:
     build_only: true
-    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y
+    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y DCONFIG_CHIP_LIB_SHELL=y DCONFIG_CHIP_COMMISSIONABLE_DEVICE_TYPE=y 
+      DCONFIG_CHIP_ROTATING_DEVICE_ID=y DCONFIG_CHIP_DEVICE_TYPE=10
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-      - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
```

Comment:
Remove nrf7002?

Reply:
Yes, so it will not build it, the platform is still in list of "allowed" ones. There is new configuration sample.matter.lock.smp_dfu_ffs_wifi for nRF7002 DK that will be used in system tests so we will have it covered. The documentation will build with correct supported platforms list.

Reply:
I mean don't we want to remove it from allowed platforms since we have a dedicated dfu+ffs+wifi group? Not to build it in nightly tests, either.

Reply:
I see, I will remove it.